src/ducks/apps/components/Sections/components/Filters.jsx

```diff
@@ -10,18 +10,17 @@ import SettingIcon from 'cozy-ui/transpiled/react/Icons/Setting'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
-import { useEffect } from 'react'
 
 const { BarRight } = cozy.bar
 
 const Filters = ({ filter, onFilterChange }) => {
   const anchorRef = useRef()
   const [menuDisplayed, setMenuDisplayed] = useState(false)
-  const [appUnderMaintenance, setAppUnderMaintenance] = useState(true)
 
-  useEffect(() => {
-    setAppUnderMaintenance(filter.underMaintenance == undefined)
-  }, [filter.underMaintenance])
+  const appUnderMaintenance = useMemo(
```

Comment:
nit: as we are editing this section, can we rename `appUnderMaintenance` variable with something like `areAppUnderMaintenanceShown` or `hasAppUnderMaintenanceFilter`?

With the current name my first understanding was that we were checking if an actual app was under maintenance, I didn't realize that it was about a filter.